Lib/test/test_dis.py

```diff
@@ -1414,7 +1414,7 @@ def jumpy():
         if i > 6:
             break
     else:
-        print("I can haz else clause?")
+        print("else")
```

Comment:
The long sentences stick out more and help signpost the diffs when you make a change that modifies the output. I think you have space in the table for slightly longer strings.

Reply:
done